configuration.nix

```diff
@@ -38,13 +38,28 @@
   security.pam.services.login.makeHomeDir = true;
   security.pam.services.sshd.makeHomeDir = true;
 
-  users.mutableUsers = false;
-  users.users.npmccallum = {
-    isNormalUser = true;
-    extraGroups = [ "wheel" ];
-    openssh.authorizedKeys.keys = [
-      "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBIoZP5bZedmrj/lidLkKXhvZwwl9Pj5VxLV22nXhkijt7UJhSUX/rOV4Kg/wmR5ptMjGyE4PPSHmCEzXvQnpyMU= nathaniel@mccallum.life"
-    ];
+  users.users = {
+    npmccallum = {
+      isNormalUser = true;
+      extraGroups = [ "wheel" ];
```

Comment:
should probably also be in the sgx group, and only use wheel privileges for other operations.